dev_cmds/utils/paths.ts

```diff
@@ -1,11 +1,12 @@
-import { resolve } from "path";
+import path, { resolve } from "path";
 
 const repoRoot = resolve(__dirname, "..", "..");
 
 const packagesDir = resolve(repoRoot, "packages");
 const devcmdCliPackageDir = resolve(packagesDir, "devcmd-cli");
 const devcmdPackageDir = resolve(packagesDir, "devcmd");
 
+const devPath = path.join(repoRoot, 'dev');
```

Comment:
minor: for consistency, please import the `join` function like `resolve` instead of importing the entire `path` object :)